client/starwhale/api/_impl/model.py

```diff
@@ -223,57 +227,69 @@ def _starwhale_internal_run_ppl(self) -> None:
             raise FieldTypeOrValueError("context.dataset_uris is empty")
 
         # TODO: user custom config batch size, max_retries
-        for ds_uri in self.context.dataset_uris:
-            _uri = URI(ds_uri, expected_type=URIType.DATASET)
-            consumption = get_dataset_consumption(
-                dataset_uri=_uri, session_id=self.context.version
-            )
-            loader = get_data_loader(_uri, session_consumption=consumption)
-
+        for uri_str in self.context.dataset_uris:
+            _uri = URI(uri_str, expected_type=URIType.DATASET)
+            ds = Dataset.dataset(_uri)
```

Comment:
Dataset.dataset function can handle uri string type automatically.

Reply:
this section also need URI's function : `index_with_dataset=[
                                f"{_uri.object}_#@#_{row.index}" for row in rows
                            ],`

Reply:
ok